haskell/src/Lorentz/Contracts/Stablecoin.hs

```diff
@@ -2,7 +2,91 @@
 -- SPDX-License-Identifier: MIT
 
 module Lorentz.Contracts.Stablecoin
-    (
-    ) where
+  ( Parameter
+  , Storage
+  , stablecoinPermissionsDescriptor
+  , stablecoinTokenMetadata
+  ) where
+
+import Lorentz
+import qualified Lorentz.Contracts.Spec.FA2Interface as FA2
+import Lorentz.Contracts.Spec.FA2Interface (OperatorTransferMode(..), OwnerTransferMode(..), SelfTransferMode(..))
+import qualified Lorentz as L
+import Util.Named
+
+-- Parameter
+type Parameter = FA2.Parameter
```

Comment:
Btw, it's not the actual parameter our contract has. In `stablecoin.tz` the first (topmost) entrypoint is `balance_of`, but here it is `transfer`. It works because we never use `Parameter` directly, apparently we only use it as a type parameter for `TAddress` to satisfy some entrypoint-related constraints. I think it's ok, but should be mentioned in a comment (that it's not the actual parameter, rather some isomorphic one).

Reply:
I have removed the type alias now. And instead just case the contract to FA2 parameter in when passing the originated contract to the FA2 tests. I have added the comment at that location.